haskell-compile.el

```diff
@@ -62,6 +63,11 @@ The `%s' placeholder is replaced by the current buffer's filename."
   :group 'haskell-compile
   :type 'boolean)
 
+(defcustom haskell-compile-color nil
```

Comment:
Since you're talking about a follow-up change later, let's just be bold and do this unconditionally.

Reply:
ok, will do!